lib/models/ApplicationFunction.dart

```diff
@@ -0,0 +1,20 @@
+class ApplicationFunction {
+  String methodName;
+  String message;
+  List<String>? parameters;
```

Comment:
These can all be `final`. If you do that, you can remove the getters below, since `final` public instance variables are gettable but not settable.

Reply:
I missed this. Good to know!